modules/auth/service.js

```diff
@@ -0,0 +1,16 @@
+const User = require("./model");
+
+const create = async (body) => User.create(body);
+
+const getByEmail = async (email) => User.findOne({email});
+
+const update = async (id, body) => User.findByIdAndUpdate(id, body, { new: true });
+
+const getById = async (id) => User.findById(id);
```

Comment:
przesun powyzej `getByEmail`. Nie zmieni to nic w wykonaniu kodu, ale bedzie bardziej uporzadkowanie. Rowniez: przesun obydwa `get...` powyzej `create`. Staraj sie zachowac te kolejnosc: Read, Create, Update, Delete. Znowu, jest to utarta konwencja.